includes/data.php

```diff
@@ -1532,6 +1532,51 @@ function pods_mb_substr( $string, $start, $length = null, $encoding = null ) {
 
 }
 
+/**
+ * Evaluate tags like magic tags but through pods_v and sanitize for SQL with an empty double quote if needed.
+ *
+ * @since 2.7.23
+ *
+ * @param string|array|object $tags String to be evaluated.
+ *
+ * @return string Evaluated string.
+ *
+ * @see pods_evaluate_tag
+ */
+function pods_evaluate_tags_sql( $tags ) {
+	// The temporary placeholder we will use.
+	$placeholder = '__PODS__TMP__EMPTY_VALUE__';
+
+	// Evaluate the magic tags.
+	$evaluated = pods_evaluate_tags( $tags, true, $placeholder );
+
+	$find = array(
+		'= ' . placeholder,
+		'=' . placeholder,
+		$placeholder,
+	);
+
+	$replace = array(
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 5509d360c8e68b1384896b81b060d4cd</sub>